File: Array/Easy/isArraySorted.py
def isArraySorted(arr):
    for i in range(0, len(arr)-1):
        if(arr[i] > arr[i+1]):
            return False
    return True

arr = [1,2,3,4,3,5]
if(isArraySorted(arr)):
    print("The array is sorted")
else:
    print("The array is not sorted")

"""
Time Complexity: O(N)
Space Complexity: O(1)
"""

File: Array/Easy/js/isArraySorted.js
function isArraySorted(arr){
    for(let i=0;i<arr.length;i++){
        if(arr[i]>arr[i+1]){
            return false;
        }
    }
    return true
}

arr = [1,2,3,4,3,5]
if(isArraySorted(arr)){
    console.log("The array is sorted")
} else{
    console.log("The array is not sorted")
}